apps/site/assets/ts/helpers/departureInfo.tsx

```diff
@@ -123,11 +135,54 @@ const isCommuterRail = (departureInfo: DepartureInfo): boolean => {
   return departureInfo?.routeMode === "commuter_rail";
 };
 
+const DefaultWrapper: React.FunctionComponent<unknown> = ({ children }) => (
+  <li>{children}</li>
+);
+/**
+ * From a list of `DepartureInfo[]`, generate a list of `<DisplayTime />`.
+ * Each `<DisplayTime />` will recieve the `isCR` and `targetDate` props.
+ *
+ * This function can optionally
+ * - wrap each `<DisplayTime />` in a specified wrapper element, `WrapperEl`
+ * - truncate the list to a desired `listLength` value
+ */
+const departuresListFromInfos = (
+  departureInfos: DepartureInfo[],
+  isCR: boolean,
+  targetDate?: Date,
+  listLength?: number,
+  WrapperEl: typeof DefaultWrapper = DefaultWrapper
```

Comment:
Why is this needed?

Reply:
@kotva006 Does the comment on the line above this function declaration help answer this at all? What details are you looking for?

Reply:
I understand what it does, I just don't understand the point.  As this functionality doesn't appear to be used on this pull request

Reply:
@kotva006 looks like we commented at the same time - see my second comment